gossipd/routing.c

```diff
@@ -45,19 +45,24 @@ struct pending_node_announce {
 #define TOKENS_PER_MSG 24
 #define TOKEN_MAX (24 * 4)
 
-static bool ratelimit(const struct routing_state *rstate,
-		      u8 *tokens, u32 prev_timestamp, u32 new_timestamp)
+static u8 update_tokens(const struct routing_state *rstate,
+			u8 tokens, u32 prev_timestamp, u32 new_timestamp)
 {
-	u64 num_tokens;
+	u64 num_tokens = tokens;
```

Comment:
what is a token here?

Reply:
It's an exciting new way to raise capital!

Wait, no, it's https://en.wikipedia.org/wiki/Token_bucket